ldk/node/src/hostClients/networkService.ts

```diff
@@ -0,0 +1,14 @@
+export interface HttpRequest {
+  url: string;
+  method: string;
+  body: string | Uint8Array;
```

Comment:
This is a good first step; I think I'll probably end up adding support for providing objects to be JSONified, but this needn't be implemented here